internal/command/metrics/machine_metrics.go

```diff
@@ -0,0 +1,64 @@
+package metrics
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+	tailor "github.com/yitsushi/file-tailor"
+)
+
+type MachineMetrics struct {
+	Namespace string             `json:"Namespace"`
+	MachineID string             `json:"MachineID"`
+	Data      FirecrackerMetrics `json:"Data"`
+}
+
+type FirecrackerMetrics map[string]map[string]int64
+
+func (mm *MachineMetrics) ToPrometheus() string {
+	output := []string{}
+	labels := fmt.Sprintf("namespace=\"%s\",id=\"%s\"", mm.Namespace, mm.MachineID)
+
+	for prefix, group := range mm.Data {
+		for key, value := range group {
+			output = append(output, fmt.Sprintf("%s_%s{%s} %d", prefix, key, labels, value))
+		}
+	}
+
+	return strings.Join(output, "\n")
+}
+
+func getMachineMetrics(namespace, id string) MachineMetrics {
+	machineMetrics := MachineMetrics{
+		Namespace: namespace,
+		MachineID: id,
+		Data:      FirecrackerMetrics{},
+	}
+
+	filePath := fmt.Sprintf(
+		"/var/lib/flintlock/vm/%s/%s/firecracker.metrics",
```

Comment:
This is insecure, I know, will fix it. (and we already have code to generate this path)